utils/mail/templateWaitlist.ts

```diff
@@ -0,0 +1,105 @@
+                <style>
+                    body {
+                    font-family: "Poppins";
+                    }
+                </style>
+            </head>
+            <body style="background-color: #fff; padding: 30px; position: absolute">
+                <p style="font-size: 30px"><img src=https://images.squarespace-cdn.com/content/5e83092341f99d6d384777ef/1592547010897-WF00319AKLJCVGJZC3ZK/sdc+logo+with+name+alt.png?content-type=image%2Fpng
+                    style="width: 250px; padding-bottom: 10px; color: #0c53a0" alt="SDC Logo"/></p>
+                <span style="font-style: normal;
+                    font-weight: bold;
+                    font-size: 24px;
+                    line-height: 36px;">A spot is available for ${name}!</span>
+                <p style="font-style: normal;
+                    font-weight: normal;
+                    font-size: 16px;
+                    line-height: 24px;">This is an automatic notification for waitlisted users of the following class:</p>
+                <div style="display: flex; flex-direction: row">
+                    <div
+                        style="
+                            width: 100px;
+                            height: 100px;
+                            background: url('${imageLink}');
+                            display: flex;
+                            background-size: cover;
+                        "
+                    ></div>
+                    <div style="padding-left: 10px">
+                        <span style="width: 375px; font-weight: Bold; font-size: 16px"
+                            >${name}</span
+                        ><br /><span
+                            style="width: 30px; color: rgba(115, 115, 115, 1); font-size: 14px"
+                            >${weekdayToString(
+                                classWeekday,
+                                language,
+                            )} ${convertToShortTimeRange(
+        startTimeMinutes,
+        durationMinutes,
+    )}</span
+                        ><br /><span
+                            style="width: 30px; color: rgba(115, 115, 115, 1); font-size: 14px"
+                        >${start} to ${end} 
+                        </span>
+                    </div>
+                </div>
+                <br></br>
+                <p>Use this link to register for the class:</p>
+                <a href="${
+                    process.env.NEXTAUTH_URL
+                }/program-details/${classId}">${
+        process.env.NEXTAUTH_URL
+    }/program-details/${classId}</a>
+                <br></br>
+                <br></br>
+                <p>Use this link to remove yourself from the class waitlist:</p>
+                <a href="${process.env.NEXTAUTH_URL}/class">${
+        process.env.NEXTAUTH_URL
+    }/class</a>
+                <br></br>
+                <br></br>
+                <p>Regards, Social Diversity for Children</p>
+            </body>
+            `;
+};
```

Comment:
<img width="1167" alt="Screen Shot 2021-11-14 at 7 29 16 PM" src="https://user-images.githubusercontent.com/45214377/141705863-cd6cb3b4-0b94-4885-b058-267187c59732.png">

for reference, this is what the current email looks like. i'm not sure if this was the best way to word everything lol let me know your thoughts!
@bekahma @vedantdsn @MatoPlus

Reply:
something to note is that this email is sent _**every time**_ an unenrollment occurs -- that means if eg a parent unenrolls 3 children at once, everyone will get 3 emails ...

rickson and i were talking about just wording it in a way so that it doesn't seem spammy and is like... indicating that a spot has opened up ?? so 3 emails indicates 3 spots have opened up ?? could this be worded better to align with that / should we think about implementing something to prevent multiple emails within a short time range?

Reply:
Yea curious as to what the designers think. I feel like we could get away by changing the main header in the mail to `A spot opened up for <Class>!`

Reply:
i was thinking of that phrase but i wasn't sure if there'd be any ambiguity on what it meant (is "opened up" as easily understandable to non-native speakers?), and there seems to be no intent on translating ? (considering our target audience + languages)

Reply:
That's a very good point. We could ask for translations actually (especially if the designers are set on something), if not, I think it's fine to leave it as you worded as well